### DISCUSSION TITLE
Request for Comments: Beyond 1.0

### DISCUSSION THREAD

**Discussion by coredoesdev** [ZTL-ARTCC/scheddy#7] [Ideas]
**Request for Comments**
Scheddy will be released on Saturday, November 16, 2024.
If you have general ideas for how to improve, drop them here and we can discuss!
**Have a _specific_ feature request?** [Post here instead](https://github.com/ZTL-ARTCC/scheddy/issues/new?assignees=&labels=C-feature-request%2C+needs-triage%2C+not+approved&projects=&template=feature-request.md&title=)
**Reporting a bug?** [Post here instead](https://github.com/ZTL-ARTCC/scheddy/issues/new?assignees=&labels=C-bug%2C+needs-triage%2C+not+approved&projects=&template=bug-report.md&title=)

**RFC Status**

- [x] Tracking issue not required
- [ ] Initial comment period <-- you are here
- [ ] Planning and development period
- [ ] Final comment period

**Comment by jahaight** [comment]
I am unable to change the session type durations. I can add a new session type but so not see anywhere to edit how long each session lasts.

**Reply by coredoesdev** [reply]
#10

**Comment by jahaight** [comment]
https://gyazo.com/c26467c12f74f8eb83185ae67c1fa186 - Can we list these in the booking page like this?

**Reply by coredoesdev** [reply]
#11
